### PR TITLE
docs: Restructured adoption view #699

### PR DESCRIPTION
**Pull request by jkbquabeck**
## WHAT

This PR brings in a restuctured Adoption View for the Connector KIT and is linked to issue #699.
Contents were restructured, terminology section moved from Development View into Adoption View and notice section was added.

## WHY

The old adoption view was not structured enough and differed from other KITs.

### CODE REVIEW

File: docs/kit/Adoption View/00_adoption-view.md [hunk 1]
```diff
@@ -1,27 +1,58 @@
...
+
+### Mission
+
+Collaboration needs trust. Within Catena-X trust is built on an open-source architecture and components designed to support full sovereignity.
+The Connector KIT provides the single source of information and technology to perform trustful data exchange within Catena-X. The Connector KIT bundles the interaction patterns, relevant standards, APIs, and reference implementations for developers.
+
+## Concept
+
+Trust, interoperability, and data sovereignty are the objectives and values for secure and sustainable peer-to-peer data exchange between organizations and companies. The claim is data sovereignty: Whoever provides data to the Data Space retains control and decides individually who is involved in the data exchange, how, when, where and under what conditions. A corresponding concept was developed in the context of [Gaia-X][gaiax-url] and the [International Data Space Association][idsa-url]. __The essential software component is the EDC Connector.__
```
Comment (bcronin90) [inline docs/kit/Adoption View/00_adoption-view.md:25]: The C in EDC is short for Connector. So I'd prefer to call it the EDC or the Eclipse Dataspace Connector.

Comment (bcronin90) [reply docs/kit/Adoption View/00_adoption-view.md:25]: Comes up a few times in this file.

File: docs/kit/Adoption View/00_adoption-view.md [hunk 2]
```diff
@@ -31,13 +62,106 @@ The main difference between the EDC and the previous connectors of the [IDSA][id
...
+
+#### Contract Definition
+
+By combining [`Assets`](#data asset) and Policies, Contracts for data offerings are defined. These Contracts need to be accepted by consuming participants (Connectors) for the data exchange to take place.
+
+#### Contract offer
+
+The contract offer is a representation of the [`ContractDefinition`](#contract-definition) for a specific consumer and serves as protocol for a data transfer object (DTO) for a particular contract negotiation. If a data consumer wants to conclude a binding data exchange contract based on the terms of a Data Offer, the data consumer can communicate such desire to the data provider by referencing to a specific Data Offer. Under German law, this constitutes a binding offer by the data consumer. For now, the dtaa consumer only has the option to accept all terms of a Data Offer (or not). The Data Exchange Process does not yet provide for the data consumer to make an offer that deviates from the terms of a Data Offer as set by the data provider.
```
Comment (bcronin90) [inline docs/kit/Adoption View/00_adoption-view.md:121]: I don't know whether the reference to German law is appropriate here, as the EDC hopes to be international in scope.

**Comment by stephanbcbauer** [comment]
@jkbquabeck Are we just waiting for the architecture picture? Or is something else missing? thx

**Comment by bcronin90** [comment]
> @jkbquabeck Are we just waiting for the architecture picture? Or is something else missing? thx

Once my comments are addressed, I mean to approve. Though, that doesn't mean much, considering I'm no Contributor.

**Comment by jkbquabeck** [comment]
Since the contents of the adoption view moved to the github.io repo, do we still need to merge it here?

**Pull request closed without merging**